### PR TITLE
OCPBUGS-39083: fix switching namespaces in policies

### PR DESCRIPTION
**Pull request by upalatucci**
Remove the `useEffect` and set the `activeTabKey` in the Tab `onSelect` .

The `useEffect` was there to change the `activeTabKey` on navigation change but we can even extract that from the location pathname at every render. 

**Before**

https://github.com/user-attachments/assets/4cbb2f62-2883-4127-b983-d9c480e2f095

**After**

https://github.com/user-attachments/assets/289728eb-ae29-4c9b-8b85-cae7f57c1412

### CODE REVIEW

File: src/views/networkpolicies/list/utils.ts [hunk 1]
```diff
@@ -9,3 +11,13 @@ export const getActiveKeyFromPathname = (pathname: string) => {
 
   return TAB_INDEXES.NETWORK;
 };
+
+export const getUrlFromTab = (tabIndex: number | string, namespace: string): string => {
```
Comment (metalice) [inline src/views/networkpolicies/list/utils.ts:15]: getNetworkPolicyUrlTab

Comment (pcbailey) [reply src/views/networkpolicies/list/utils.ts:15]: getNetworkPolicyURLTab -- Acronyms should be all uppercase.

File: start-console.sh [hunk 2]
```diff
@@ -64,7 +64,7 @@ if [ -x "$(command -v podman)" ]; then
         podman run --pull=always --rm --network=host --env-file <(set | grep BRIDGE) $CONSOLE_IMAGE
     else
         BRIDGE_PLUGINS="${runningPlugins["podman"]}"
-        podman run --pull=always --rm -p "$CONSOLE_PORT":9000 --env-file <(set | grep BRIDGE) $CONSOLE_IMAGE
+        podman run --platform=linux/x86_64 --pull=always --rm -p "$CONSOLE_PORT":9000 --env-file <(set | grep BRIDGE) $CONSOLE_IMAGE
```
[the remark this answers is not in the document]
Comment (upalatucci) [reply start-console.sh:67]: i'll solve that

**Review by metalice — APPROVED** [review]

**Pull request merged**